app/templates/flatpages/privacy-policy.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block head %}
+<title>Privacy Policy | HackItForward</title>
+{% endblock %}
+
+{% block body %}
+<space class="large"></space>
+<div class="content" style="margin-bottom: 90px;">
+  <h1 style="text-align: center; margin-bottom: 40px;">Privacy Policy</h1>
+  <ul>
+    <li>HackItForward stores a small amount of data such as your username, your profile picture and other user-generated data on the platform when you create an account.</li>
```

Comment:
I really like privacy policies with tables that specifies what piece of data is collected and what it's used for.